README.md

```diff
@@ -3,10 +3,10 @@
 This is a Template for any Python based project, it contains what Project Thoth and the AI CoE need:
 
 1. GitHub defaults and Templates for issues
-2. configuration for Coala and Black (code formating)
-3. basic configuration for Zuul
-4. configuration for Thoth (stage environment, Red Hat VPN only)
-5. if you are writing a Python module, Kebechet could manage releases of your packages for you
-6. if credentials are provided, Python module releases could be published to PyPI
+2. Configuration for Coala and Black (code formating) are done using [`prow`](https://github.com/kubernetes/test-infra/tree/master/prow) with `pre-commit` and `mypy`.
+3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
+4. We have a public service configuration now accessible for everyone.
+5. If you are writing a Python module, Kebechet could manage releases of your packages for you
```

Comment:
let's link the bot for 5. and 6. https://github.com/marketplace/khebhut

Reply:
Done.
Can anything else be improved?

Reply:
that was all on my mind, thanks @tlegen-k! @fridex @goern @harshad16 something more to add?